zeq/hbp/detail/imageRawRGBA8.fbs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015, Human Brain Project
+//                     Grigori Chevtchenko <grigori.chevtchenko@epfl.ch>
+
+namespace zeq.hbp;
+
+table ImageRawRGBA8
```

Comment:
I am still ok if "raw" not in the name, but at a point we may have to define some proper Image class.